modules/pel/peltool/peltool.py

```diff
@@ -354,7 +358,7 @@ def deleteAllPELs(path: str) -> None:
         break
 
 
-def process_pelID(pelID: str) -> str:
+def processId(pelID: str) -> str:
```

Comment:
Suggestion: I think pelID was meant for EID so I think we could rename it as `id`.

Reply:
Agreed. instead of id i will mention as pid. because id() is a built-in function